jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/jsonb/JsonbTransientAnnotationQuickFix.java

```diff
@@ -8,21 +8,40 @@
 * SPDX-License-Identifier: EPL-2.0
 *
 * Contributors:
-*     IBM Corporation - initial API and implementation
+*     IBM Corporation, Adit Rada - initial API and implementation
 *******************************************************************************/
 package org.eclipse.lsp4jakarta.jdt.core.jsonb;
 
-import org.eclipse.lsp4jakarta.jdt.codeAction.proposal.quickfix.RemoveAnnotationConflictQuickFix;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.lsp4jakarta.jdt.codeAction.proposal.RemoveMultipleAnnotations;
 
 /**
  * Quick fix for removing @JsonbTransient annotations when more than
- * one occur in a class
+ * one occur in a class.
+ * The getCodeActions method is overridden in order to make sure that
+ * we return our custom quick fixes. There will be two quick fixes given
+ * to the user: (1) either remove @JsonbTranient or (2) remove all other
```

Comment:
spelling, should be "either remove `@JsonbTransient`"

Reply:
Done